### PR TITLE
update VersionQuery to use Amber V2 tables

### PR DESCRIPTION
**Pull request by yetti**
@scoen @jrixon @m-r-c @weijunnla @shuangzhou @ninhnguyen 

Had to make a couple `private` static fields in `AmberVertexMapper` and `AmberQueryBase` public so I could reuse it instead of duplicating the same code.

I'm also not sure if `VersionQueryTest` ever tested correctly, given that we've created a book with 15 pages, then we're deleting a page from the book on line 106. Anyhow, I've updated the test.

### CODE REVIEW

**Comment by yetti** [comment]
Will close this for now until I fix the other test that's breaking. :(

**Comment by yetti** [comment]
Let's try this again...

@scoen @jrixon @m-r-c @weijunnla @shuangzhou @ninhnguyen please have a look

**Comment by nlabuilder** [comment]
Build finished.

File: src/main/java/amberdb/graph/AmberVertexMapper.java [hunk 1]
```diff
@@ -20,12 +20,17 @@
     
     private AmberGraph graph;
     
-    private static Set<String> skipProps = new HashSet<>();
+    public static Set<String> skipProps = new HashSet<>();
     static {
     	skipProps.add("id");
     	skipProps.add("txn_start");
     	skipProps.add("txn_end");
     	skipProps.add("state");
```
Comment (yetti) [inline src/main/java/amberdb/graph/AmberVertexMapper.java]: @jrixon Needed to add these properties to the set of properties to skip. The mapper was adding the temporary table fields to the Vertex properties after saving SoundFiles and causing the `AmberDao::suspendIntoFlatVertexTable()` method to fail with an SQL error.

**Comment by nlabuilder** [comment]
Build finished.

File: src/main/java/amberdb/graph/AmberVertexMapper.java (hunk 1 above)
Comment (jrixon) [inline src/main/java/amberdb/graph/AmberVertexMapper.java]: Yep, looks fair enough.

**Pull request merged**